client/src/components/dom/Auth/Auth.tsx

```diff
@@ -1,34 +1,38 @@
-import * as React from "react";
```

Comment:
In general,

`import * as React from 'react';`

is preferred over 

`import {...} from 'react'`;

as we don't need to commit extra changes whenever a new hook/function enters the mix. If I need a `useRef` I can just `React.useRef()` rather than add a whole new import. This also makes it clear which hooks come from where, and ensures we don't get duplicate

```
import {x} from 'react';
import {y} from 'react';
```

which is all too common.